src/Moryx.AbstractionLayer.Products.Endpoints/PartialSerialization.cs

```diff
@@ -26,14 +26,6 @@ static PartialSerialization()
             _serialization = new EntrySerializeSerialization();
         }
 
-        /// <summary>
-        /// Creates a new <see cref="PartialSerialization{T}"/>  instance
-        /// </summary>
-        [Obsolete("Use serialization with containers instead")]
```

Comment:
Please check for all uses of `PossibleValuesSerialization`, All of them should use the container + service collection, not sure if we might missed to put some obsolete flags somewhere